docs/userguide.rst

```diff
@@ -221,3 +221,31 @@ were deployed during ``init``, you can run:
 .. code-block:: console
 
    $ reana-cluster down
+
+Delete interactive sessions
+---------------------------
+
+Interactive sessions that were not closed after work have been finished leave
+active Kubernetes objects (pod, service and ingress) running in the Reana cluster.
+These objects might need to be deleted from time to time. Please note that
+interactive session pod and service are linked to the ingress object. If ingress
+object gets deleted other twos will be deleted automatically.
+
+To delete open interactive sessions we suggest to use following commands:
```

Comment:
I would maybe remove the part we say "we suggest to" and just state that this is the current way of deleting sessions.